app/models/project.rb

```diff
@@ -0,0 +1,3 @@
+class Project < ApplicationRecord
+  validates :title, :date, :description, presence: { message: "must be given please" }
```

Comment:
you don't need specify message if you like default one. There is a reason to manually set it?